REQUIRE

```diff
@@ -1 +1,2 @@
 julia 0.6
+Compat
```

Comment:
prior to 0.38 won't support uninitialized https://github.com/JuliaLang/Compat.jl/commit/c72b467cf9693308fa99eddba9abfcc378f25147

Reply:
Thanks, I'll add that into https://github.com/staticfloat/SHA.jl/pull/52

Reply:
did you already tag this with a too-loose requirement? are copyto and GC even in a Compat release yet?

Reply:
`copyto!()` and `GC` aren't merged into `master` yet, so they're not tagged.  I did already tag this with a too-loose requirement.  Unfortunately CI doesn't catch things like this.  :)

Reply:
You can amend it in metadata. Also see RequirementVersions.jl. You folks have gotten worse lately about merging and releasing changes to widely-used packages without even waiting for anyone else to review. I'm glad I don't actually rely on Julia any more but it's unfortunate for those who do.